gkeys/gkeys/config.py

```diff
@@ -85,6 +85,7 @@ def _add_gkey_defaults(self):
         self.defaults['verify-keyring'] = 'gentoo'
         self.defaults['verify-nick'] = 'gkeys'
         self.defaults['verify-seeds'] = {}
+        self.defaults['days_limit'] = 180
```

Comment:
REALLY?  you want to start bitching (because that is what it will sound like) at people 6 months before or after a key expires?  that may have been needed for your initial testing to get a decent amount of positives.  But please configure a normal default.

Please refer to the spec_check default of 30 days which could also be made a configured option and re-use your days_limit config value.